src/test/java/uk/gov/hmcts/reform/notificationservice/service/NotificationServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.notificationservice.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.notificationservice.data.NotificationRepository;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class NotificationServiceTest {
+
+    @Mock
+    private NotificationRepository notificationRepository;
```

Comment:
no need for `private` I guess

Reply:
why not? it is recommended to use the lowest possible access level

Reply:
it's a test

Reply:
it's variable not a test. but i don't see a point to discuss here